lib/components/narrative/connected-trip-details.js

```diff
@@ -13,6 +13,8 @@ const TripDetails = styled(TripDetailsBase)`
 
 const mapStateToProps = (state, ownProps) => {
   return {
+    defaultFare: state.otp.config.itinerary?.defaultFare || undefined,
+    fareKeyNameMap: state.otp.config.itinerary?.fareKeyMap || {},
```

Comment:
Also rename `state.otp.config.itinerary?.fareKeyMap` -> `.fareKeyNameMap`

Reply:
Updated in a32088189205a527c35064378e6ef614edd5b98e